datasets/tess.yaml

```diff
@@ -0,0 +1,24 @@
+Name: Transiting Exoplanet Survey Satellite (TESS)
+Description: |
+  The Transiting Exoplanet Survey Satellite (TESS) is a two-year survey that will discover exoplanets in orbit around bright stars.
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Monthly
+Tags:
+  - astronomy
+  - aws-pds
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files in a 'Requester Pays' bucket
+    ARN: arn:aws:s3:::stpubdata
```

Comment:
These data are in `stpubdata/tess` is it acceptable to have two entries in this registry with the same bucket (the Hubble data are in `stpubdata/hst`)?

Reply:
Yep, I think that's fine. I would just list the ARN as `arn:aws:s3:::stpubdata/tess` and update to `arn:aws:s3:::stpubdata/hst` in the other file.